client/sttp-client/src/main/scalajs/sttp/tapir/client/sttp/SttpClientInterpreterExtensions.scala

```diff
@@ -8,6 +8,8 @@ import scala.concurrent.Future
 
 trait SttpClientInterpreterExtensions {
 
+  def sttpClientOptions: SttpClientOptions = SttpClientOptions.default
```

Comment:
since this is always extended by `SttpClientInterpreter`, we might use self-types here and avoid the additional definiton of `sttpClientOptions`.

Sth like: `trait SttpClientInterpreterExtensions { this: SttpClientInterpreter =>`